app/v1/models/request/NameModel.scala

```diff
@@ -101,15 +109,41 @@ object NameModel {
     }
   }
 
-  implicit val reads: Reads[NameModel] = (
-    titlePath.readNullable[String].filter(titleValidationError)(validateTitle) and
-      forename.readNullable[String].filter(nameValidationError("forename"))(validateName(_, "forename")) and
-      secondForename.readNullable[String].filter(nameValidationError("second forename"))(validateName(_, "secondForename")) and
-      surname.read[String].filter(nameValidationError("surname"))(validateName(_, "surname")) and
-      startDate.readNullable[DateModel] and
-      endDate.readNullable[DateModel] and
-      typePath.read[String].filter(typeValidationError)(validateType)
-    ) (NameModel.apply _)
+  private[models] def validateDateAsPriorDate(earlierDate: Option[DateModel], laterDate: Option[DateModel]) = {
+    (earlierDate, laterDate) match {
+      case (Some(earlierDateModel), Some(laterDateModel)) =>
+        val earlierModelAsDate = LocalDate.parse(earlierDateModel.dateString, DateTimeFormatter.ofPattern("dd-MM-yyyy"))
+        val laterModelAsDate = LocalDate.parse(laterDateModel.dateString, DateTimeFormatter.ofPattern("dd-MM-yyyy"))
+        Logger.warn("[NameModel][validateDateAsPriorDate] The provided earlierDate is after the laterDate.")
+        earlierModelAsDate.isBefore(laterModelAsDate) || earlierModelAsDate.isEqual(laterModelAsDate)
+      case _ => true
+    }
+
+  }
+
+  private def todayDateAsDateModel = Some(DateModel(LocalDate.now(ZoneId.of("UTC")).format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))))
+
+  implicit val reads: Reads[NameModel] = for {
+    title <- titlePath.readNullable[String].filter(titleValidationError)(validateTitle)
+    foreName <- forenamePath.readNullable[String].filter(nameValidationError("forename"))(validateName(_, "forename"))
+    secondForename <- secondForenamePath.readNullable[String].filter(nameValidationError("second forename"))(validateName(_, "secondForename"))
+    surname <- surnamePath.read[String].filter(nameValidationError("surname"))(validateName(_, "surname"))
+    startDate <- startDatePath.readNullable[DateModel].filter(dateNonPriorError)(validateDateAsPriorDate(_, todayDateAsDateModel))
```

Comment:
The validation needs to be that the startDate is before the endDate. It can't be equal to it.